app/src/main/res/xml/provider_paths.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <paths>
     <external-path path="Android/data/com.amaze.filemanager/" name="package_root" />
-    <external-path path="." name="storage_root" />
+    <root-path path="." name="storage_root" />
```

Comment:
Would you please leaving a comment in file to reference this issue or the StackOverflow link, for documentation purposes?

Reply:
Sounds good, let me know what you think about the added comment.